scripts/azure-templates-bootstrapper.yml

```diff
@@ -15,7 +15,7 @@ parameters:
   condition: succeeded()                        # whether or not to run this template
   shouldPublish: true                           # whether or not to publish the artifacts
   configuration: $(CONFIGURATION)               # the build configuration
-  buildExternals: ''                            # the build number to download externals from
+  buildExternals: '4127355'                            # the build number to download externals from
```

Comment:
this looks like a broken indent

Reply:
This is a temporary hack to speed up the CI builds. Usually it takes over an hour to build all the native bits, but for this PR, we can just download the last build's artifacts.